### ISSUE TITLE
Expand Tag and Category Management

### ISSUE THREAD

**Issue by da2ce7** [torrust/torrust-index-gui#283]
Tags and Categories should be able to be managed in a convenient way:

Including the following functions:

Tags:
1. Create
2. Update
3. Duplicate
4. Delete

Categories:
1. Create
2. Update
3. Merge Into
4. Remove

**Comment by josecelano** [comment]
Relates to: https://github.com/torrust/torrust-index-archive/issues/104

**Comment by da2ce7** [comment]
@josecelano I'm recreating all the issues, etc, in planning to delete the  `torrust-index` repo.

**Comment by josecelano** [comment]
> @josecelano I'm recreating all the issues, etc, in planning to delete the  `torrust-index` repo.

So, are you planning to create user features in this repo, and then open new issues in the backend if they are needed? For me it's ok to start from a user's needs and then split it into subtasks that can affect more than one repo.

**Comment by da2ce7** [comment]
Yes. User Story Issues are Here. We can link relevant (back-end) issues from here.
